docs/ru/getting-started/example-datasets/nyc-taxi.md

```diff
@@ -267,7 +267,7 @@ FROM trips
 Это занимает 3030 секунд со скоростью около 428 тысяч строк в секунду.
 Для более короткого времени загрузки, можно создать таблицу с движком `Log` вместо `MergeTree`. В этом случае загрузка отработает быстрее, чем за 200 секунд.
 
-Таблица заняла 126 Гб дискового пространства.
+Таблица заняла 126 Гб пространства хранения.
```

Comment:
This is worse.

Reply:
Disk = HDD (hard disk drive), SSD (solid state drive) is not a disk. Moreover, we can attach a volume on top of several drives (RAID, LUN from SAN).